macros/edr/tests/test_exposure_schema_validity.sql

```diff
@@ -0,0 +1,58 @@
+{% test exposure_schema_validity(model, exposures, node, columns) %}
```

Comment:
Maybe add a comment that `exposures`, `nodes` and `columns` exist as parameters only for dependency injection from the integration tests (right?)